class ConflictError(Exception):
    pass


class AuthorizationError(Exception):
    pass
